packages/url_launcher/url_launcher/README.md

```diff
@@ -72,7 +72,12 @@ for examples of other queries.
     <action android:name="android.intent.action.DIAL" />
     <data android:scheme="tel" />
   </intent>
-  <!-- If your app emails -->
+  <!-- If your sends SMS messages -->
+  <intent>
+    <action android:name="android.intent.action.SENDTO" />
+    <data android:scheme="smsto" />
```

Comment:
Is this a typo? I've only ever seen `sms` as the scheme.

Reply:
From the Android docs:

> 
> If you want to ensure that your intent is handled only by a text messaging app (and not other email or social apps), then use the ACTION_SENDTO action and include the "smsto:" data scheme. For example:
>    ` <data android:scheme="smsto" />`
>

Reply:
TIL!